api/proto/config/netconfig.proto

```diff
@@ -48,6 +48,11 @@ message NetworkAdapter {
 
   // firewall
   repeated ACE acls = 40;
+
+  // access port vlan id
+  // app interface with access vlan id of zero will be treated as trunk port
+  // valid vlan id range: 1 - 4094
```

Comment:
Nit: max is 4093.
In addition some documentation claims 1 and 4093 is reserved. But I don't think we need to care about that.

Reply:
will change the comment

Reply:
DOne